src/fu-engine.c

```diff
@@ -905,8 +912,24 @@ fu_engine_modify_bios_settings(FuEngine *self, GHashTable *settings, GError **er
 				    (const gchar *)key);
 			return FALSE;
 		}
-		if (!fu_engine_modify_single_bios_setting(self, key, value, error))
-			return FALSE;
+		if (!fu_engine_modify_single_bios_setting(self, key, value, &error_local)) {
+			if (g_error_matches(error_local, FWUPD_ERROR, FWUPD_ERROR_NOTHING_TO_DO)) {
+				g_debug("%s", error_local->message);
+				continue;
+			} else {
```

Comment:
i don't think you need the }else{ here

Reply:
also, maybe cherry pick this one to main

Reply:
ack